test/TestUtilities/Test.Utility/Protocol/V3PackageSearchMetadataFixture.cs

```diff
@@ -74,6 +74,8 @@ public MockPackageSearchMetadata()
 
             public Uri ReadmeUrl => null;
 
+            public Uri ReadmeFileUrl => null;
```

Comment:
I was thinking there should be a few more tests that need this new property added to existing tests.

Some new tests may be needed as well, like this Local metadata for IconUri, for example: checked:https://github.com/NuGet/NuGet.Client/blob/e2f077a9a062e8190b3482e29786a758182b33fb/test/NuGet.Core.Tests/NuGet.Protocol.Tests/LocalPackageSearchMetadataTests.cs#L62

Reply:
Meh, added a unit test to match the one you linked. It looks like there's only one test class for LocalPackageSearchMetadata.